### ISSUE TITLE
Handle Validation onBlur

### ISSUE THREAD

**Issue by happylinks** [Astrocoders/reform#57]
Hi, do you have any advice on how to handle validation onBlur with this library?

I was thinking of a workaround with handling onBlur myself and then doing something like SetFocussedField("email"), and checking that before showing the validation message. But probably it could be done in a nicer way.

I'm willing to make a PR if this is a feature that more people would like :)

**Comment by grsabreu** [comment]
The library itself does not provide yet a onBlur handler, we recommend bringing your own handler for now.

That is interesting to have indeed, we didn't had this use case before (of keeping the focused field state).

probably something like?
```ocaml
<Form etc>
 ...({ focusedField: FormParams.fields, setFocusedField: FormParams.fields => unit, handleChange } => {
 })
</Form>
```

**Comment by happylinks** [comment]
Yeah, I think focusedField could work. It would let the user decide themselves what they want to do with that info.
My use-case is only showing the validation message if there is an error and the user is already done with the input (not focused anymore).
I'll see if I can set something up soon.

**Comment by MarcelCutts** [comment]
I just came across this exact use case also @happylinks! 

I also want my user to be able to lose focus of an email field before they're attacked by an error message, because their first keystroke will trigger that error regardless.

**Comment by MarcelCutts** [comment]
My thought-solution was decoupling validation from onChange but this way is probably more elegant.

**Comment by happylinks** [comment]
This is now released in 5.1.1-beta.6. Closing this issue :)
